ignite/contrib/handlers/tqdm_logger.py

```diff
@@ -113,7 +113,9 @@ def _reset(self, pbar_total):
             **self.tqdm_kwargs
         )
 
-    def _close(self, engine):
+    def _close(self, engine, log_handler, event_name):
+        if not self.pbar:
+            self.pbar = log_handler(engine, self, event_name, close=True)
```

Comment:
I do not quite understand the purpose of this line. Why we can not just ignore `self.pbar.close()` if `self.pbar is None` ?

Reply:
@vfdev-5 the reason I did this was so that the user could still see the progress bar status in case of a sudden close/exception raised. 

I agree, there is no need to add unnecessary complexity to this.